game/src/ecs/components/ai/transition/ProtectOnAttack.java

```diff
@@ -0,0 +1,70 @@
+package ecs.components.ai.transition;
+
+import ecs.components.HealthComponent;
+import ecs.components.MissingComponentException;
+import ecs.components.PlayableComponent;
+import ecs.entities.Entity;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Lets AI protect an entity
+ *
+ * <p>Implements an AI that protects a specific entity with a HealthComponent if the hero afflicting
+ * damage to it. Entity will stay in fight mode
```

Comment:
```suggestion
 * Implements an AI that protects a specific entity with a HealthComponent, if the hero dealt damage to it.
 * <p> Entity will stay in fight mode once entered.
```
Um konsistent mit der Javadoc von ProtectOnApproach zu bleiben